packages/next-ui/ActionCard/ActionCardListForm.tsx

```diff
@@ -36,6 +39,8 @@ export function ActionCardListForm<
     ...other
   } = props
   const RenderItem = render as React.FC<ActionCardItemRenderProps<ActionCardItemBase>>
+  const [show, setShow] = React.useState(false)
+  const showMoreAfter = 4
```

Comment:
showMoreAfter can not be hardcoded here and must be an opt-in functionality. This should be passed from the caller side `<ActionCardListForm showMoreAfter={4}/>`